Feat individual deploy

notanmay opened the pull request.

notanmay committed:
feat: :sparkles: add launch config

Commit:
feat(Individual Deploy): :sparkles: Added command for deploying

jappurohit041 committed:
update

jappurohit041 committed:
feat(Individual Deploy): :sparkles: added the test-service and updated logic

jappurohit041 committed:
style(Individual Deploy): :art: improved the structure of code and added TO-DO and lifecycle events

notanmay committed:
feat: :sparkles: build array from lambda ARNs

jappurohit041 committed:
feat(Individual Deploy): :sparkles: invidividual step Function deployment over

jappurohit041 committed:
refactor(Individual Deploy): :art: removing extra es-lint disabled comments

KillDozerX2 commented on lib/deploy/deploy.js:
```diff
@@ -0,0 +1,102 @@
+
+module.exports = {
+  /*
+   * Will get stack resources and create object and append in obj
+   * with key as ARN Name and value as ARN Value
```
Please add the description that you explained to me.

jappurohit041 replied:
Added detailed description

KillDozerX2 commented on lib/deploy/deploy.js:
```diff
@@ -0,0 +1,102 @@
+    function iterateStates(obj) {
+      const data = {};
+      data.States = {};
+      for (const key in obj) {
+        if (key === 'States') {
+          const subStates = obj[key];
```
Please use [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)

jappurohit041 replied:
Used and modified logic along with simplifying calling.

notanmay committed:
test(Individual Deploy): :white_check_mark: add test cases for individual deploy

jappurohit041 committed:
refactor(Individual Deploy): :art: modified documentation, recursion function logic

modified the recursion function for iterating over the states, detailed documentation of function

jappurohit041 committed:
docs(Individual Deploy): :memo: updated the documentation of readme.md file
